tests/src/test/scala/whisk/core/limits/ActionLimitsTests.scala

```diff
@@ -269,4 +269,29 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers {
             }
     }
 
+    it should "limit the length of HTTP req/resp bodies for debug messages to 1000 bytes" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+            val name = "largeFileAction"
+            val debugMsg = "Body excedes 1000 bytes and will be truncated"
+            val bodyMsg = "ENDUNIQUE"   //  Message that should be truncated
+
+            val actionCode = new File(s"$testActionsDir${File.separator}$name.js")
+            actionCode.createNewFile()
```

Comment:
Why create the file for each test run instead of having one in `dat/actions`? You can use `echo.js` and send a large string as a parameter to it.